inc/cloudwatch_error_handler/namespace.php

```diff
@@ -48,6 +48,24 @@ function error_handler( int $errno, string $errstr, string $errfile = null, int
 	return false;
 }
 
+/**
+ * When script execution ends, send any buffered errors to CloudWatch.
+ */
+function send_buffered_errors_on_shutdown() {
+	$errors = $GLOBALS['altis_cloudwatch_error_handler_errors'];
```

Comment:
Is this always set?

Reply:
Mm I'll add a guard, just in case. We don't want a recursive error!